# ActivateObstacle.srv
int64 obstacleNumber
---
string response